comparisons/validate.C

```diff
@@ -1519,23 +1519,23 @@ void validateEvents(TString step, TString file, TString refFile, TString r="RECO
       tbr="TotemFEDInfos_totemRPRawToDigi_RP_"+recoS+".obj";
       if (checkBranchOR(tbr, true)){
         plotvar(tbr+"@.size()");
-        plotvar(tbr+".getFEDId()");
-        plotvar(tbr+".getOptoRxId()");
-        plotvar(tbr+".getFSize()");
+        plotvar(tbr+".fedId()");
```

Comment:
new method names should be added, not replaced.
This is to keep this script functional on the old releases.

Reply:
Thanks @slava77 ! Added these methods back.